lib/pure/strutils.nim

```diff
@@ -254,6 +313,34 @@ func toUpperAscii*(s: string): string {.rtl, extern: "nsuToUpperAsciiStr".} =
     doAssert toUpperAscii("FooBar!") == "FOOBAR!"
   toImpl toUpperAscii
 
+func removePunctAscii*(s: string): string {.rtl, extern: "nsuRemovePunctAscii".} =
```

Comment:
this is bloat; remove both removePunctAscii and removeSpecialAscii, they're best replaced by a more generic and flexible (and efficient!) alternative that would filter in-place based on a set